docker/mysql_tests/scripts/base_tests/full_xtrabackup_test_with_ranges.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e -x
+
+. /usr/local/export_common.sh
+
+export WALE_S3_PREFIX=s3://mysqlfullxtrabackupbucket
```

Comment:
Add the new bucket [here](https://github.com/wal-g/wal-g/blob/14a0e5b4cc45457fee047a93199c08abda07c768/docker-compose.yml#L53)

Reply:
+